xfce-extra/xfce4-whiskermenu-plugin/xfce4-whiskermenu-plugin-1.6.0.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils gnome2-utils
+
+DESCRIPTION="Alternate application launcher for Xfce"
+HOMEPAGE="https://gottcode.org/xfce4-whiskermenu-plugin"
+SRC_URI="https://github.com/gottcode/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE=""
+
+RDEPEND="
+	virtual/libintl:=
+	x11-libs/gtk+:2=
+	xfce-base/exo:=
+	xfce-base/garcon:=
+	xfce-base/libxfce4ui:=
+	xfce-base/libxfce4util:=
+	xfce-base/xfce4-panel:="
```

Comment:
Those are slot operators, actually. They make it possible to keep things working when SONAME actually changes, rather the usual nonsense of waiting for the milk to be spilt.